File: frontend/src/app/models/beneficio.ts
export interface Beneficio {
  id?: number;
  nome: string;
  descricao?: string;
  valor: number;
  ativo: boolean;
  version?: number;
}

export interface CreateUpdateBeneficioDto {
  nome: string;
  descricao?: string;
  valor: number;
  ativo?: boolean;
}

export interface TransferRequest {
  fromId: number;
  toId: number;
  amount: number;
}

File: backend-module/src/main/java/com/example/ejb/Beneficio.java
package com.example.ejb;

import lombok.Data;

import java.io.Serializable;
import java.math.BigDecimal;

@Data
public class Beneficio implements Serializable {
    private Long id;
    private String nome;
    private String descricao;
    private BigDecimal valor;
    private Boolean ativo;

}